help/destinations/catalog/advertising/liveramp-distribution.md

```diff
@@ -11,15 +9,14 @@ The [!DNL LiveRamp - Distribution] connection enables you to activate audiences
 
 >[!IMPORTANT]
 >
->This destination connector and documentation page are created and maintained by LiveRamp. For any inquiries or update requests, please contact LiveRamp directly [here](mailto:example@email.com).
+>This destination connector and documentation page are created and maintained by LiveRamp. For any inquiries or update requests, please contact LiveRamp directly [here](mailto:adobertcdp@liveramp.com).
 
 ## Supported destinations {#supported-destinations}
 
 [!DNL LiveRamp - Distribution] currently supports audience activation to the following platforms:
 
 * [!DNL 4C Insights]
```

Comment:
Please add the corresponding links from this list to the headings further below.